src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/Table.java

```diff
@@ -86,6 +92,8 @@ public class Table<T> extends Content {
     private HtmlStyle gridStyle;
     private final List<Content> bodyRows;
     private HtmlId id;
+    private static String regex = "<(?:a|area|button|input|object|select|textarea)\\b";
+    private static final Pattern pattern = Pattern.compile(regex);
```

Comment:
Please inline `regex` into the pattern initial value, and use a more descriptive name than just `pattern`.

Reply:
Updated changes as suggested. Thank you/